### DISCUSSION TITLE
Node.js API doc support search

### DISCUSSION THREAD

**Discussion by F3n67u** [nodejs/help#4959] [Q&A] [answered]
Is there any reason why https://nodejs.org/api/ doesn't support search? I want to search some keywords across nodejs API doc multiple times, but currently, the API site doesn't support search.

I tried searching if there was any similar feature request in the issue list, but I don't find it. I am wondering there is must some reason why our API doc does not have a search feature. So I ask first here.

**Comment by aduh95** [comment, marked as answer]
Duplicate of https://github.com/nodejs/node/issues/36412 it seems.

**Reply by F3n67u** [reply]
Yes. Thanks for your answer.
